wallet/src/test/java/com/android/identity_credential/wallet/logging/EventLoggerTest.kt

```diff
@@ -30,28 +38,35 @@ class EventLoggerTest {
     }
 
     @Test
-    fun testDeleteEntries() {
-        // Start logging events
+    fun testAddAndDeleteEntries() {
         activityLogger.startLoggingEvents()
 
-        // Add an MdocPresentationEntry to the logger
         activityLogger.addMDocPresentationEntry(
             documentId = "doc123",
             sessionTranscript = "sessionTranscript".toByteArray(),
             deviceRequestCbor = "request data".toByteArray(),
-            deviceResponseCbor = "response data".toByteArray()
+            deviceResponseCbor = "response data".toByteArray(),
+            requesterType = requesterType,
+            shareType = shareType,
         )
 
-        // Retrieve the entries for the given documentId
         val entries = activityLogger.getEntries("doc123")
-
-        // Ensure that entries were added correctly
         assertTrue(entries.isNotEmpty(), "Entries should not be empty after adding an event")
 
-        // Delete the entries retrieved
         activityLogger.deleteEntries(entries)
-
-        // Check that the storage is empty after deletion
         assertTrue(mockStorage.enumerate().isEmpty(), "Storage should be empty after deleting all entries")
     }
-}
+
+    companion object {
+        @JvmStatic
+        @Parameterized.Parameters
+        fun data(): Collection<Array<Any>> {
+            val data = mutableListOf<Array<Any>>()
+            data.add(arrayOf(EventLogger.Requester.Anonymous(), EventLogger.ShareType.SHARED_IN_PERSON))
```

Comment:
We should test named and unknown requesters.

Reply:
Some how the comments came in together. Here are the replies:
For the date, added todo.
For the comment, removed.
Added unit tests coverage.